test/e2e/tests/transaction/edit-gas-fee.spec.js

```diff
@@ -7,6 +7,14 @@ const {
 const { withFixtures, unlockWallet, WINDOW_TITLES } = require('../../helpers');
 const FixtureBuilder = require('../../fixture-builder');
 
+const PREFERENCES_STATE_MOCK = {
+  preferences: {
+    showFiatInTestnets: true,
+  },
+  // Enables advanced details due to migration 123
+  useNonceField: true,
```

Comment:
Why do we now need to open advanced details on these tests?

Reply:
By default, you only see the fiat network fee, but these tests also assert the native / token fee.